wicket-kendo-ui-culture/pom.xml

```diff
@@ -25,6 +25,12 @@
 			<artifactId>wicket-jquery-ui-core</artifactId>
 			<version>6.20.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.wicket-jquery-ui</groupId>
+			<artifactId>wicket-kendo-ui</artifactId>
+			<version>6.20.1-SNAPSHOT</version>
+			<type>bundle</type>
```

Comment:
Hi @paulbors,

The `maven-bundle-plugin` usage has been introduced by #104 to support OSGi. But I should say that I am also not an expert in this area (not to mention I don't like it)... All artifactd are declared as `<packaging>bundle</packaging>` and OSGi-users seems to be happy with this (even I do not really understand all the concerns...)

Personally, I don't have any build issue, but of course I probably have the setup that works... But if you find the correct-not-too-heavy project/pom setup thats complies with OSGi boundling, you win a.... _(well, gift to be defined)_ ! :)